docs/reference/aggregations/bucket/daterange-aggregation.asciidoc

```diff
@@ -61,6 +61,43 @@ Response:
 --------------------------------------------------
 // TESTRESPONSE[s/\.\.\./"took": $body.took,"timed_out": false,"_shards": $body._shards,"hits": $body.hits,/]
 
+==== Missing Values
+
+The `missing` parameter defines how documents that are missing a value should be treated.
+By default they will be ignored but it is also possible to treat them as if they had a value.
+This is done by adding a set of fieldname : value mappings to specify default values per field.
```

Comment:
Please wrap these to 80-columns. 😄

Reply:
Most of the documentation is wrapped at 140 characters like the code. I was not aware that we had taken a decision to change this to 80 characters.